tests/unit/bokeh/plotting/test__graph.py

```diff
@@ -17,7 +17,7 @@
 #-----------------------------------------------------------------------------
 
 # Bokeh imports
-from bokeh.models import ColumnDataSource, Scatter
+from bokeh.models import ColumnDataSource, Scatter, Circle, MultiLine
```

Comment:
from bokeh.models import (
    Circle,
    ColumnDataSource,
    MultiLine,
    Scatter,
)
```suggestion
from bokeh.models import (
    Circle,
    ColumnDataSource,
    MultiLine,
    Scatter,
)
```
extra imports caused an `isort` format change